CompilerSource/JDI/src/API/context.cpp

```diff
@@ -153,7 +153,7 @@ void context::copy(const context &ct)
 }
 void context::swap(context &ct) {
   if (!parse_open and !ct.parse_open) {
-    { register definition_scope* gs = ct.global;
```

Comment:
there's likely a fix for this in upstream jdi so look at @JoshDreamland repo and see what hes changed